legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -292,3 +344,28 @@ def validate_correction_name_request(filing: Dict, corrected_filing: Dict) -> Op
         return msg
 
     return None
+
+
+def validate_pdf(file_key: str):
```

Comment:
just remembered we need to check that the file isn't encrypted

Reply:
ah geez, not sure how to do that.

Reply:
probly super easy.  let me try and find the code.

Reply:
just use `pdf_reader.isEncrypted` to check

Reply:
Thats way too simple.. can't be real :)

Reply:
yeah i know hey 😀

Reply:
Added 👍

Reply:
do you think "File must be unencrypted" is more ideal for the validation message?

Reply:
Yes, thats a little for sure! Updated 👍

Reply:
looks good!